runtime/compiler/optimizer/J9RecognizedCallTransformer.hpp

```diff
@@ -85,6 +85,22 @@ class RecognizedCallTransformer : public OMR::RecognizedCallTransformer
     *     \endcode
     */
    void process_java_lang_Class_IsAssignableFrom(TR::TreeTop* treetop, TR::Node* node);
+   /** \brief
+    *     Transforms java/lang/StringCoding.encodeASCII(B[B)[B into a compiler intrinsic for codegen acceleration (i.e encodeASCII symbol).
+    *
+    *  \param treetop
+    *     The treetop which anchors the call node.
+    *
+    *  \param node
+    *     The call node representing a call to java/lang/StringCoding.encodeASCII(B[B)[B which has the following shape:
+    *
+    *     \code
+    *     acall <java/lang/StringUTF16.toBytes([CII)[B>
```

Comment:
I don't think this is the right signature

Reply:
Fixed here: 18163e58a18208669debb6efd2261b8e744b8660